code/game/objects/items/bodybag.dm

```diff
@@ -124,7 +124,7 @@
 	if(mob_to_stuff.stat != DEAD) //Only the dead for bodybags.
 		return FALSE
 	var/mob/living/carbon/human/human_to_stuff = mob_to_stuff
-	if((!check_tod(human_to_stuff) || issynth(human_to_stuff)) && human_to_stuff.is_revivable())
+	if((check_tod(human_to_stuff) && human_to_stuff.is_revivable()) || issynth(human_to_stuff))
```

Comment:
You could revive a headless synth with this change.

Reply:
This does not change revival code, this merely fixes closing bodybags in some situations

Reply:
It does change behavior, is what I mean.
Old code previews two behaviors:
1) If it's synth, check `is_revivable()` (because a headless synth can't be revived)
2) Whether synth or not, check if `undefibbable` and enough time has passed, which is the `check_tod()` macro.

New code has two different behaviors.
1) If it's synth, check always succeeds. This allows for headless synths to be revived.
2) If other cases, it needs both `is_revivable()` and `s_revivable()` to pass.

This is because `&&` is evaluated before `||`

Reply:
The old behavior is broken, though.